### PR TITLE
Add tracking for some steps

### PR DESCRIPTION
**Pull request by jleveugle**
MANAGER-2764

Add events to facilitate tracking

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/new/new.controller.js [hunk 1]
```diff
@@ -256,7 +257,16 @@ export default class PciProjectNewCtrl {
   ============================== */
 
   onNextBtnClick() {
+    const page = `public-cloud::${this.$state.current.name.replace(/\./g, '::')}`;
```
Comment (Jisay) [inline packages/manager/modules/pci/src/projects/new/new.controller.js]: `$state` is not injected in controller but in routing file

File: packages/manager/modules/pci/src/projects/new/payment/payment.controller.js [hunk 2]
```diff
@@ -2,10 +2,20 @@ import find from 'lodash/find';
...
+    $state,
+    $translate,
+    atInternet,
+    CucCloudMessage,
+    ovhPaymentMethod,
+    PciProjectNewService,
+  ) {
     // dependencies injections
     this.$q = $q;
+    this.$state = $state;
```
Comment (Jisay) [inline packages/manager/modules/pci/src/projects/new/payment/payment.controller.js]: I though that `$state` was only injected in routing file?

File: packages/manager/modules/pci/src/projects/new/new.controller.js (hunk 1 above)
Comment (jleveugle) [reply packages/manager/modules/pci/src/projects/new/new.controller.js]: right!

File: packages/manager/modules/pci/src/projects/new/payment/payment.controller.js (hunk 2 above)
Comment (jleveugle) [reply packages/manager/modules/pci/src/projects/new/payment/payment.controller.js]: No, `$state` is already injected here :)

File: packages/components/ng-uirouter-breadcrumb/rollup.config.js [hunk 3]
```diff
@@ -4,14 +4,19 @@ const config = rollupConfig({
...
-export default [
-  config.cjs(),
+const outputs = [
   config.es(),
-  config.umd({
+];
+
+if (process.env.BUILD === 'production') {
+  outputs.push(config.cjs());
+  outputs.push(config.umd({
```
Comment (marie-j) [inline packages/components/ng-uirouter-breadcrumb/rollup.config.js]: 👍

**Comment by ovh-cds** [comment]
CDS Report installation#3151.6 ✘  
* Stage 1
  * Install ✘

File: packages/manager/modules/pci/src/projects/new/new.controller.js [hunk 4]
```diff
@@ -257,6 +258,14 @@ export default class PciProjectNewCtrl {
 
   onNextBtnClick() {
     const currentStep = this.getCurrentStep();
+
+    if ((this.paymentModel.mode === 'credits' && this.paymentModel.credit.value)) {
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/new/new.controller.js]:
```suggestion
    if (this.paymentModel.mode === 'credits' && this.paymentModel.credit.value) {
```
?

File: packages/manager/modules/pci/src/projects/new/new.routing.js [hunk 5]
```diff
@@ -82,22 +82,53 @@ export default /* @ngInject */ ($stateProvider) => {
...
           }
         },
-        shouldProcessChallenge: /* @ngInject */ (getCurrentStep, newProjectInfo) => () => {
+        shouldProcessChallenge: /* @ngInject */ (
+          $state,
+          atInternet,
+          getCurrentStep,
+          newProjectInfo,
+        ) => () => {
+          const page = `public-cloud::${$state.current.name.replace(/\./g, '::')}`;
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/new/new.routing.js]: As this seems to be used quite often, maybe we could extract it to a function (at least the `replace` part)

File: packages/manager/modules/pci/src/projects/new/payment/payment.routing.js [hunk 6]
```diff
@@ -62,6 +68,13 @@ export default /* @ngInject */ ($stateProvider) => {
           // cancel project creation and redirect refresh page
           const { hiPayStatus, projectId } = $transition$.params();
           if (hiPayStatus !== 'success' && get(project, 'status') === 'creating' && projectId) {
+            const page = `public-cloud::${$transition$.to().name.replace(/\./g, '::')}`;
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/new/payment/payment.routing.js]: In the component you bind `trackingPage`, why not inject it directly here ?

**Comment by antleblanc** [comment]
Both commits 77a02fa and a6094f3 aren't related to any tracking changes.

Especially for a6094f3 which is fixing an issue also for the master branch.

What do you think about splitting them into multiple pull request?

**Comment by antleblanc** [comment]
69ee04c now solves the issues related to your following commit a6094f3.

Thanks!

File: packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.html [hunk 7]
```diff
@@ -94,14 +94,15 @@ <h1 data-translate="pci_projects_project_storages_blocks_title"></h1>
             </oui-action-menu>
 
             <extra-top>
-                <oui-button
+                <button
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.html]: Any reason not to use the `oui-button` anymore ?

Comment (FredericEspiau) [reply packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.html]: I guess so as you did it three times 😄

Comment (FredericEspiau) [reply packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.html]: I get it, it's because of the transclusion

**Pull request merged**